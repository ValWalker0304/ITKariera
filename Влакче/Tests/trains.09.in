Add 25077 Zimnitsa P 3
Add 48842 Mezdra P 3
Add 76201 Montana P 6
Add 99326 Yambol P 4
Add 43981 Tulovo P 8
Add 10196 Karnobat P 3
Add 18835 Podkova P 1
Add 64216 StaraZagora P 10
Add 87858 Sliven P 2
Next
Travel
Next
Travel
Add 19706 Shumen P 2
Add 29348 Kulata P 5
Travel
Travel
Travel
Travel
Travel
Next
Travel
Next
Travel
Next
Next
History
Travel
Travel
Next
Add 63900 Plovdiv P 5
Add 17749 Burgas P 6
Add 20563 Vratsa P 4
Add 70520 Targovishte P 7
Add 38517 Yambol P 6
History
Next
Add 40929 Ruse P 6
History
Next
History
Travel
Travel
History
History
Add 41144 Sliven P 3
Travel
Next
Travel
Travel
Add 76354 Podkova P 9
Next
Travel
History
History
Travel
Next
History
History
End