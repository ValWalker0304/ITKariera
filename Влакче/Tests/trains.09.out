87858 Sliven P 2
87858 Sliven P 2
64216 StaraZagora P 10
64216 StaraZagora P 10
29348 Kulata P 5
19706 Shumen P 2
18835 Podkova P 1
10196 Karnobat P 3
43981 Tulovo P 8
99326 Yambol P 4
99326 Yambol P 4
76201 Montana P 6
76201 Montana P 6
48842 Mezdra P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
48842 Mezdra P 3
25077 Zimnitsa P 3
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
38517 Yambol P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
40929 Ruse P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
40929 Ruse P 6
38517 Yambol P 6
38517 Yambol P 6
40929 Ruse P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
38517 Yambol P 6
40929 Ruse P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
41144 Sliven P 3
70520 Targovishte P 7
70520 Targovishte P 7
20563 Vratsa P 4
76354 Podkova P 9
76354 Podkova P 9
76354 Podkova P 9
20563 Vratsa P 4
70520 Targovishte P 7
41144 Sliven P 3
38517 Yambol P 6
40929 Ruse P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
76354 Podkova P 9
20563 Vratsa P 4
70520 Targovishte P 7
41144 Sliven P 3
38517 Yambol P 6
40929 Ruse P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
17749 Burgas P 6
63900 Plovdiv P 5
17749 Burgas P 6
76354 Podkova P 9
20563 Vratsa P 4
70520 Targovishte P 7
41144 Sliven P 3
38517 Yambol P 6
40929 Ruse P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
17749 Burgas P 6
76354 Podkova P 9
20563 Vratsa P 4
70520 Targovishte P 7
41144 Sliven P 3
38517 Yambol P 6
40929 Ruse P 6
25077 Zimnitsa P 3
48842 Mezdra P 3
76201 Montana P 6
99326 Yambol P 4
43981 Tulovo P 8
10196 Karnobat P 3
18835 Podkova P 1
19706 Shumen P 2
29348 Kulata P 5
64216 StaraZagora P 10
87858 Sliven P 2
